### PR TITLE
Add EON Mainnet

### PR DESCRIPTION
**Pull request by light-fury**
Fixes # .

Changes proposed in this pull request:
- 
- 
-

### CODE REVIEW

**Comment by light-fury** [comment]
Hi @ChaituVR 
I already checked this PR with [Snapshot](https://snapshot-networks.on.fleek.co/) following the guide.
Could you please review this PR? Since we are waiting for this to be merged to continue our development.

**Comment by ChaituVR** [comment]
Hi @light-fury we have changed our process to add a new network, kindly refer https://docs.snapshot.org/developer-guides/networks#add-a-new-network

**Comment by light-fury** [comment]
Hi @ChaituVR 
Thank you.
I just submitted the form using that portal.

**Comment by light-fury** [comment]
Hi @ChaituVR 
I am sorry but I created the form with the wrong network name.
The actual name must be `Horizen EON`.
I submitted 2 forms. The first one is with `EON` and the second one is the correct one with `Horizen EON`.
So I hope you to ignore the second form with the name `EON`.
Thank you.

File: src/networks.json [hunk 1]
```diff
@@ -1618,6 +1618,23 @@
     "start": 370457,
     "logo": "ipfs://QmWkhZYhReYyaa5pQXj32hEGxoRcBqarFMcfQScELmjYQj"
   },
+  "7332": {
+    "key": "7332",
+    "name": "EON",
```
Comment (ChaituVR) [inline src/networks.json:1623]:
```suggestion
    "name": "Horizen EON",
```

File: src/networks.json [hunk 2]
```diff
@@ -1618,6 +1618,23 @@
...
+    "shortName": "EON",
+    "chainId": 7332,
+    "network": "mainnet",
+    "testnet": false,
```
Comment (ChaituVR) [inline src/networks.json:1627]:
```suggestion
```

File: src/networks.json [hunk 3]
```diff
@@ -1618,6 +1618,23 @@
...
+    "multicall": "0xC743e4910Bdd4e5aBacCA38F74cdA270281C5eef",
+    "rpc": [
+      "https://eon-rpc.horizenlabs.io/ethv1"
```
Comment (ChaituVR) [inline src/networks.json:1630]:
```suggestion
```

File: src/networks.json [hunk 4]
```diff
@@ -1618,6 +1618,21 @@
...
+    "shortName": "EON",
+    "chainId": 7332,
+    "network": "mainnet",
+    "multicall": "0xC743e4910Bdd4e5aBacCA38F74cdA270281C5eef",
+    "rpc": [
+    ],
+    "explorer": {
+      "url": "https://eon-explorer.horizenlabs.io"
+    },
+    "start": 1,
```
Comment (ChaituVR) [inline src/networks.json:1633]:
```suggestion
    "start": 85108,
```

**Review by ChaituVR — APPROVED** [review]

**Comment by ChaituVR** [comment]
@light-fury It is live https://snapshot.org/#/?filter=networks&q=7332 🚀

**Comment by cbonaco1** [comment]
hi @ChaituVR sorry for the confusion but i see the `rpc` was removed and set to an empty array in this PR. I'm under the impression we will need that. can you please add that back, or let us know how we can update it on our end?

**Comment by ChaituVR** [comment]
Hi @cbonaco1 We don't need this anymore, We use this RPC node directly on our proxy server now :)

**Comment by cbonaco1** [comment]
okay great, thank you for the explanation!

**Pull request merged**